app/api/views/briefs.py

```diff
@@ -261,6 +265,43 @@ def create_rfx_brief():
     return jsonify(brief.serialize(with_users=False))
 
 
+@api.route('/brief/training', methods=['POST'])
+@login_required
+@role_required('buyer')
+@permissions_required('create_drafts')
+def create_training_brief():
+    try:
+        lot = lots_service.find(slug='training2').one_or_none()
```

Comment:
Should this be moved to a business file?

Reply:
updated for training